app/src/androidTest/java/org/kiwix/kiwixmobile/note/NoteFragmentTest.kt

```diff
@@ -18,18 +18,31 @@
 
 package org.kiwix.kiwixmobile.note
 
+import android.os.Build
 import androidx.test.internal.runner.junit4.statement.UiThreadStatement
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import org.junit.Before
 import org.junit.Test
 import org.kiwix.kiwixmobile.BaseActivityTest
 import org.kiwix.kiwixmobile.R
 
 class NoteFragmentTest : BaseActivityTest() {
+
+  @Before
+  override fun waitForIdle() {
+    UiDevice.getInstance(InstrumentationRegistry.getInstrumentation()).waitForIdle()
+  }
+
   @Test
   fun verifyNoteFragment() {
-    UiThreadStatement.runOnUiThread { activityRule.activity.navigate(R.id.notesFragment) }
-    note {
-      assertNoteRecyclerViewExist()
-      assertSwitchWidgetExist()
+    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
+      UiThreadStatement.runOnUiThread { activityRule.activity.navigate(R.id.notesFragment) }
+      note {
+        assertToolbarExist()
+        assertNoteRecyclerViewExist()
```

Comment:
We should remove this check. I can't find the tickets to solve this kind of problem.

Reply:
hi @gouri-panda , done